accounts/admin.py

```diff
@@ -1,3 +1,62 @@
 from django.contrib import admin
+from unfold.admin import ModelAdmin
+from django.contrib.auth.models import Group
 
 # Register your models here.
+from allauth.account.models import EmailAddress
```

Comment:
Also I think this isn't really needed, let's revert this one, just for courses is enough

Reply:
This added for adapt email adress admin with unfold styling 

when remove it 
result: 
![Screenshot 2024-02-15 at 15 18 00](https://github.com/DataTalksClub/course-management-platform/assets/69054810/84bf367e-98b3-4f48-a0cd-1b5b1afa4348)

Reply:
Why do we need a panel for it? I was hoping we could remove the entire accounts/admin.py file

Reply:
for update email address admin styling for to be as unfold styling...

Reply:
if you want hide email address and other models as (sites and socialaccount) models let me know we can unregister this models from admin 
but if you want show it we should [re-register](https://unfoldadmin.com/installation/) with unfold for apply styling